README.md

```diff
@@ -336,6 +336,14 @@ It's also possible to easily awnser a question by calling the following method w
 const isQuestionAwnsered: Boolean = await qnaI.answerQuestion(question, "Lakers should win!")
 ```
 
+To delete questions you can use the following method:
```

Comment:
Please add these points:
- the user who sent the question can delete it
- moderators can delete any message